rabbitmq/include/userver/urabbitmq/client_settings.hpp

```diff
@@ -29,6 +29,12 @@ struct AuthSettings final {
 
   /// RabbitMQs vhost
   std::string vhost = "/";
+
+  /// TLS
+  std::string client_cert_path;
```

Comment:
Let's move all these fields into an `std::optional<TlsSettings>`, I think it would be cleaner this way

Reply:
Done